City-of-Helsinki/drupal-asuntotuotanto
# SATT-117: npm package updates & cross-spawn security update

### Description

Npm package updates & cross-spawn security update


### how to test / verify

- cd public/themes/custom/asuntotuotanto
- npm i (maybe need use nvm use)
- npm run build (check that theme building goes through without errors, warnings shows)
- npm outdated - you will see that almost everything is updated, sass-loader version 16 will broke theme build
- npm audit report - shows npm report
